tests/unit_tests/test_job_builder.py

```diff
@@ -0,0 +1,88 @@
+import json
```

Comment:
do you want to put this one under the launch directory?

Reply:
I can if you prefer, I was thinking since this is testing something that will happen with all runs, it does not belong in the launch directory

Reply:
yeah... i guess you are right... i was just under the impression it's only relevant for launch, but if we actually do the build for every run so probably current location makes more sense